src/main/java/com/teamfive/disscard/DissCardController.java

```diff
@@ -101,9 +101,10 @@ public ResponseEntity<Card> addCard(@RequestBody Card card) {
         HttpHeaders headers = new HttpHeaders();
         try {
             newCard = cardService.save(card);
+            log.info("Card added successfully: {}", newCard);
         }
         catch (Exception e) {
-            log.error(e.getMessage());
+            log.error("Error while adding card: {}. Exception: {}", card, e.getMessage());
```

Comment:
Updated the logging statements to provide a more detailed message to make it easier to diagnose issues. Good logging practices lead to better observability to easily monitor performance.
https://www.slf4j.org/manual.html